automation/start_flex_aio_microk8s_demo.sh

```
@@ -0,0 +1,153 @@
+  EXT_IP=$(curl --silent --max-time 10 ipinfo.io/ip || true)
+  if [[ -z "$EXT_IP" ]]; then
+    echo "[E] Unable to determine external IP. Please provide it as the 4th argument."
+    exit 1
+  fi
+fi
+
+sudo apt-get update
+sudo snap install microk8s --classic
+sudo microk8s.status --wait-ready
+sudo microk8s.enable dns ingress hostpath-storage helm3
+mkdir -p -m 0700 "$HOME/.kube"
```

Shellcheck (0.11.0)

[warning] When used with -p, -m only applies to the deepest directory.

(SC2174)